src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+    >
+      <ec-icon
+        class="ec-summary-info__icon"
+        data-test="ec-summary-info__icon"
+        :class="{'tw-mr-12': iconName}"
+        :name="iconName"
+        :size="12"
+      />
+    </div>
+
+    <div class="ec-summary-info__content">
+      <div
+        v-if="title"
+        class="ec-summary-info__title"
+        data-test="ec-summary-info__title"
+      >
+        {{ title }}
+      </div>
+
+      <div
+        v-if="lineItems.length > 0 || hasSlot('default')"
+        data-test="ec-summary-info__content-lines"
+        class="ec-summary-info__content-lines"
+      >
+
+        <div
+          v-for="item in lineItems"
+          :key="item.text"
+          :class="[item.cssClasses]"
+          class="ec-summary-info__content-line-item"
+        >
+          <span
+            :class="[item.textCssClasses]"
+          >
+            {{ item.text }}
+          </span>
+
+          <span
+            v-if="item.iconName"
+            v-ec-tooltip="{
+              content: item.tooltipText
+            }"
+            :class="{'tw-ml-4': item.iconName}"
+          >
+            <ec-icon
+              class="ec-summary-info__content-line-item-icon"
+              :class="[item.iconCssClasses]"
+              :name="item.iconName"
+              :size="14"
+            />
+          </span>
+        </div>
+
+        <slot />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+
+import { useSlots } from 'vue';
+
+import VEcTooltip from '../../directives/ec-tooltip';
+import EcIcon from '../ec-icon';
+
+defineProps({
+  title: {
```

Comment:
There is no need to have a title property anymore, this will be a line, with preset style "secondary"(name is an example). Ordinary line will have preset name "default"

Reply:
ok, corrected. thanks